.github/workflows/deploy.yaml

```
@@ -47,11 +47,13 @@ jobs:
 
       - name: ECS 태스크 정의 업데이트
         run: |
+          # 현재 태스크 정의 가져오기
           TASK_DEFINITION=$(
           aws ecs describe-task-definition \
            --task-definition ${{ vars.ECS_TASK_DEFINITION }}
           )
-
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)